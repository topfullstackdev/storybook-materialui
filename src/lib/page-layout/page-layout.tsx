import { ReactNode } from 'react';

export interface PageLayoutProps {
  children: ReactNode;
}

export function PageLayout({ children }: PageLayoutProps) {
  return (
    <div
      css={{
        maxWidth: '1400px',
        width: '100%',
        marginLeft: 'auto',
        marginRight: 'auto',
      }}
    >
      {children}
    </div>
  );
}

export default PageLayout;
