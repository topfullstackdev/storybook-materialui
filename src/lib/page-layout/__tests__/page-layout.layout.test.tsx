import { render } from '@testing-library/react';

import { PageLayout } from '../page-layout';

describe('PageLayout', () => {
  it('should render successfully', () => {
    const { container } = render(<PageLayout>content</PageLayout>);
    expect(container).toMatchSnapshot();
  });
});
