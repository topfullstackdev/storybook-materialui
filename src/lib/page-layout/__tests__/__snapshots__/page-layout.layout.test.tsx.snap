// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout should render successfully 1`] = `
.emotion-0 {
  max-width: 1400px;
  width: 100%;
  margin-left: auto;
  margin-right: auto;
}

<div>
  <div
    class="emotion-0"
  >
    content
  </div>
</div>
`;
